src/core/data-cite.js

```diff
@@ -26,6 +26,10 @@ function requestLookup(conf) {
     let href = "";
     // This is just referring to this document
     if (key.toLowerCase() === conf.shortName.toLowerCase()) {
+      showInlineWarning(
```

Comment:
the suggestion is to replace this with just `console.warn()`

Reply:
Could it be `console.log`, as respec2html still will potentially throw if warnings exist?

Reply:
good point. Yeah, `console.log()` is better here.